core/domain/user_jobs_one_off.py

```diff
@@ -81,6 +81,43 @@ def reduce(key, version_and_exp_ids):
                     edited_exploration_ids))
 
 
+class UserAuthModelOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Perhaps: ``PopulateUserAuthModelOneOffJob`` to be more specific.

Reply:
Renamed it now.